pkg/sql/plan/function/builtin/multi/serial.go

```diff
@@ -36,7 +36,7 @@ func SerialWithSomeCols(vectors []*vector.Vector, proc *process.Process) (*vecto
 	vct := types.T_varchar.ToType()
 	nsp := new(nulls.Nulls)
 	val := make([][]byte, 0, length)
-	ps := types.NewPackerArray(length)
+	ps := types.NewPackerArray(length, proc.Mp())
```

Comment:
where is the proc.Mp() contructed?   You want to be sure that this proc.Mp has proper configuration, that is, it has enough small obj cache.

Reply:
The configuration item of GuestMmuLimit comes from the configuration file of mo:
guest mmu limitation.  default: 1 << 40 = 109951162777